From d77c711277bf075ce0545c19b322f040c49c778a Mon Sep 17 00:00:00 2001
From: Alexander Kanavin <alex.kanavin@gmail.com>
Date: Thu, 30 Apr 2015 12:19:13 +0300
Subject: [PATCH] Install example binary needed for core-image-clutter

Signed-off-by: Zhai Edwin <edwin.zhai@intel.com>

Upstream-Status: Inappropriate [configuration]

Imported from clutter-gst-1.8 package, git://git.openembedded.org/openembedded,
commit ae28ee3f7a060b9e0d13154a84f2444a98490b5b, updated for clutter-gst 3.0

Signed-off-by: Tomas Frydrych <tomas@sleepfive.com>
Signed-off-by: Alexander Kanavin <alex.kanavin@gmail.com>
---
 examples/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/examples/Makefile.am b/examples/Makefile.am
index bfbc1b5..09655e3 100644
--- a/examples/Makefile.am
+++ b/examples/Makefile.am
@@ -1,6 +1,6 @@
 NULL = #
 
-noinst_PROGRAMS = camera-player video-player video-sink video-sink-navigation video-content
+bin_PROGRAMS = camera-player video-player video-sink video-sink-navigation video-content
 
 AM_CPPFLAGS = -I$(top_srcdir) \
 		$(MAINTAINER_CFLAGS) \
-- 
2.1.4

